pos-print/src/main/java/com/clouway/pos/print/adapter/http/HttpBackend.java

```diff
@@ -76,4 +71,41 @@ public void stop() throws Exception {
     server.stop();
   }
 
+  private MongoClient establishMongoDbConnection(List<String> hosts) {
```

Comment:
It may be established in main instead of here. 

It's not a good idea persistence logic to go in the HttpBackend class. Also the CommandCLI will be removed as dependency as DB related params are not going to be required anymore and just port could be passed directly to the HttpBackend.